crates/valence_text/src/tests.rs

```diff
@@ -64,7 +64,7 @@ fn selector() {
     let deserialized = Text::from_str(&serialized).unwrap();
     assert_eq!(
         serialized,
-        r##"{"selector":"foo","separator":{"text":"bar","color":"red","bold":true}}"##
```

Comment:
this looks unrelated, should this be in this pr?

Reply:
This was caught in a clippy lint when I ran it.

Reply:
okay 👍 
LGTM then